pls_cli/please.py

```diff
@@ -142,16 +142,16 @@ def language(lang: str) -> None:
         else:
             settings['language'] = lang
             Settings().write_settings(settings)
-            # Here function Dowload quotes 
-        
+            # Here function Dowload quotes
```

Comment:
@guedesfelipe i am reformat the code, the download part would go here, but i really don't have much idea how to query the current branch to be able to download the quotes, can you give me a hint?

Reply:
Sure! I will take a look at the code and help you with that as soon as possible

Reply:
Sorry for the delay, I was busy these days and I still haven't been able to make an example code, but I was thinking of doing the part of downloading the translated files like this:
https://github.com/Textualize/rich/blob/master/examples/downloader.py

Or with Requests library...